client/settings/digital-wallets/index.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardBody,
+	CardDivider,
+	CheckboxControl,
+} from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+const DigitalWallets = () => {
+	const [ isEnabled, setIsEnabled ] = useState( false );
+	const [ sectionsStatus, setSectionsStatus ] = useState( {
+		checkout: true,
+		productPage: true,
+		cart: true,
+	} );
+
+	const makeHandleSectionStatusChange = ( section ) => ( status ) => {
+		setSectionsStatus( ( oldStatuses ) => ( {
+			...oldStatuses,
+			[ section ]: status,
+		} ) );
+	};
+
+	return (
+		<Card className="digital-wallets">
+			<CardBody size="large">
+				<CheckboxControl
+					checked={ isEnabled }
+					onChange={ setIsEnabled }
+					label={ __(
+						'Enable digital wallets & express payment methods',
+						'woocommerce-payments'
+					) }
+					help={ interpolateComponents( {
```

Comment:
You might notice some differences in spacing and styling of the help text with the PR and the designs in Figma
![Screen Shot 2021-04-28 at 8 55 56 AM](https://user-images.githubusercontent.com/273592/116416028-9bf5ac00-a7ff-11eb-9ac1-d6ac37e61c3d.png)
vs
![Screen Shot 2021-04-28 at 8 56 40 AM](https://user-images.githubusercontent.com/273592/116416115-b16ad600-a7ff-11eb-9501-a3e98a009079.png)

The difference is with the Gutenberg help text styles and the version of Gutenberg being used. UX is fine with this, as the PR stands. Otherwise we might be continuously making changes aiming at a moving target.
Eventually the Figma designs will align with the Gutenberg components.